hana/test/hana-functions.test.js

```diff
@@ -0,0 +1,50 @@
+const cds = require('../../test/cds')
```

Comment:
Should we add this function definition into the other databases? 
Then the test can be move here:
https://github.com/cap-js/cds-dbs/blob/45bcdcf30ed103c1db8283d30ce5d84a7748a1d5/test/compliance/functions.test.js#L339-L348

Reply:
I initially started there but the other databases don't support the argument

Reply:
https://github.com/cap-js/cds-dbs/pull/909 but I don't like the different behavior on the dbs

Reply:
It is a good finding that the databases have different response formats. The purpose of the cal-functions is to define them in a way that they response in the correct format. Most likely adding a `cast` or `element` to the original func can apply the correct output converter for each database.